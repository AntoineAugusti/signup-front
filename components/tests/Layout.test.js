import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import Layout from '../Layout';


describe('components | Layout', () => {
	describe('render', () => {
		const renderer = ReactTestRenderer.create(
			<Layout />
		);

		it('should be defined', () => {
			expect(renderer).toBeDefined();
		});
		it('should match snapshot', () => {
			expect(renderer).toMatchSnapshot();
		});
	});
});
