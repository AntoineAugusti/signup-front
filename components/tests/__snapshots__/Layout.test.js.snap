// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | Layout render should match snapshot 1`] = `
<div
  style={
    Object {
      "border": "1px solid #DDD",
      "margin": 20,
      "padding": 20,
    }
  }
>
  <div>
    <a
      href="/"
      onClick={[Function]}
      style={
        Object {
          "marginRight": 15,
        }
      }
    >
      Home
    </a>
    <a
      href="/about"
      onClick={[Function]}
      style={
        Object {
          "marginRight": 15,
        }
      }
    >
      About
    </a>
  </div>
</div>
`;
